test/com/facebook/buck/apple/testdata/prebuilt_apple_framework_static_swift/make_prebuiltlib.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+#
+# Helper file to regenerate the prebuilt framework for different swift versions
```

Comment:
@robbertvanginkel Can we not create a prebuilt Obj-C framework so that there's no need to recompile as Swift versions change? That would still be able to test whether the framework path is passed to the Swift compiler, right?

Reply:
Yes, very true and a much better idea. I guess I was so deep in the swift stuff I didn't even consider this. I'll make the change.